azure-pipelines/main-pipeline.yml

```diff
@@ -0,0 +1,228 @@
+# The idea is to be inclusive of all source directories. (I.E. Commerce, Accounts, Eureka, Gateway, Inventory, Config, and Base(?))
+# And using the variable to set a condition that will set pipeline variables variables that will control the flow of the pipeline.
+
+trigger:
+
+  branches:
+    include: 
+    - experimental-pipeline
+
+pr:
+  branches:
+    include:
+    - main
+    - staging
+    - development
+    - feature_*
+    
+
+# Parameters and Variables to help with naming and calling variables, as well as providing a group of objects to loop over in later jobs.
+variables:
+    
+    org: '1054-daniel-felleman-revature-swag-shop'
+    projectKey: '1054-Daniel-Felleman-Revature-Swag-Shop_RevatureShopBackend'
+    projectName: 'RevatureShopBackend'
```

Comment:
it's not sufficiently clear that these are sonarcloud organization, sonar project key, sonar project name.

Reply:
Good point, these are not specific.